src/index.js

```diff
@@ -28,6 +28,15 @@ module.exports = function(session) {
         throw new Error('No dataset provided to Datastore Session.');
       }
       this.kind = options.kind || 'Session';
+      if (
+        options.expirationMs &&
+        (typeof options.expirationMs !== typeof 0 || options.expirationMs < 0)
+      ) {
+        throw new Error(
+          'invalid value for option expirationMs: must be an integer greater than 0'
+        );
+      }
+      this.expirationMs = parseInt(options.expirationMs, 10) || 0;
```

Comment:
I tend to use `Number(variable)` these days, which has a simple algorithm that doesn't require the base:

https://www.ecma-international.org/ecma-262/10.0/index.html#sec-tonumber

Reply:
I have no objections, changing.

Reply:
Actually, looking at this again, I think `parseInt` is slightly better because we do not expect this variable to be a float. While I don't think a float would break this commit, it's possible to add confusion in a future version.

But I defer to the judgement of you project maintainers.

Reply:
@chrishiestand 👍 reasonable point, feel free to stick with `parseInt`.